target/linux/generic/files/drivers/platform/mikrotik/Kconfig

```diff
@@ -15,4 +15,10 @@ config MIKROTIK_RB_SYSFS
 	help
 	  This driver exposes RouterBoot configuration in sysfs.
 
+config NVMEM_LAYOUT_MIKROTIK
```

Comment:
Could we please set or is unset this in the configs that use `CONFIG_MIKROTIK`. Interrupts kernel config in build for the new symbol currently.
```
git grep CONFIG_MIKROTIK=y
target/linux/ath79/mikrotik/config-default:CONFIG_MIKROTIK=y
target/linux/ipq40xx/mikrotik/config-default:CONFIG_MIKROTIK=y
target/linux/ramips/mt7621/config-6.6:CONFIG_MIKROTIK=y
```

Reply:
Done